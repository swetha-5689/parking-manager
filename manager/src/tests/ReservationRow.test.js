import React from 'react';
import ReservationTableRow from '../ReservationRow';
import renderer from 'react-test-renderer';

const Reservation = {
    custFName: 'Neha',
    custLName: 'Nelson',
    type: 'Guaranteed',
    car: 'E82RTY',
    email: 'neha@example.com',
    username: 'n_nelson',
    resID: 129652,
    Paid: false,
  };
/* This test uses Date.now() as the default time, so snapshots may be out of date.
This will cause the test to fail. Everything else renders correctly. */
it('renders everything correctly except for time', () => {
    const tree = renderer.create(<ReservationTableRow obj = {Reservation}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
