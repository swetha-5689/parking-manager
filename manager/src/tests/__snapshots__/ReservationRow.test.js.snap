// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders everything correctly except for time 1`] = `
<tr
  data-testid="row"
>
  <td>
    Neha
  </td>
  <td>
    Nelson
  </td>
  <td>
    n_nelson
  </td>
  <td>
    Guaranteed
  </td>
  <td>
    false
  </td>
  <td>
    <time
      dateTime={"2020-04-03T06:44:39.522Z"}
    >
      April 3rd 2020, 2:44:39
    </time>
     - 
    <time
      dateTime={"2020-04-03T06:44:39.522Z"}
    >
      April 3rd 2020, 2:44:39
    </time>
  </td>
  <td>
    <time
      dateTime={"2020-04-03T06:44:39.523Z"}
    >
      April 3rd 2020, 2:44:39
    </time>
  </td>
  <td>
    129652
  </td>
  <td>
    <button
      className="btn btn-danger btn-sm"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Delete
    </button>
    <button
      className="btn btn-primary btn-sm"
      disabled={false}
      type="button"
    >
      Bill
    </button>
  </td>
</tr>
`;
